testing/main_test.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/magefile/mage/sh"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestSetup tests if Kibana can be run against the current registry
+// and the setup command works as expected.
+func TestSetup(t *testing.T) {
+
+	err := os.Chdir("environments")
+	if err != nil {
+		t.Error(err)
+	}
+
+	// Make sure services are shut down again at the end of the test
+	defer func() {
+		err = sh.Run("docker-compose", "-f", "snapshot.yml", "-f", "local.yml", "down", "-v")
```

Comment:
This spins up the Stack + Registry but pulls in master. Instead it should build the local registry and run it.